teste2e/author_submit_and_watch_extrinsic_test.go

```diff
@@ -99,10 +96,27 @@ func TestChain_SubmitExtrinsic(t *testing.T) {
 		panic(err)
 	}
 
-	extEnc, err := types.EncodeToHexString(ext)
+	sub, err := api.RPC.Author.SubmitAndWatchExtrinsic(ext)
 	if err != nil {
 		panic(err)
 	}
-
-	fmt.Printf("%#v\n", extEnc)
+	defer sub.Unsubscribe()
+
+	timeout := time.After(1000 * time.Second)
```

Comment:
Can we reduce the timeout? Sounds like we should be safe with a much lower limit.

Reply:
Again, very good find. I indeed forgot to change this back, set it that high for debugging. Set it to 10 seconds to give it 1 block and a bit of wiggle room to finish.